lib/howl/sys.moon

```diff
@@ -20,22 +22,27 @@ env = setmetatable {}, {
 find_executable = (name) ->
   return File(name).exists if File.is_absolute(name)
 
-  path = env['PATH']
+  if is_flatpak
+    stdout, stderr = Process.execute {'which', name}
```

Comment:
`execute` returns `stdout`, `stderr` and the process object. Use the latter and `.successful` instead of relying on the `stderr` to be available.